source/game.c

```diff
@@ -2210,7 +2237,16 @@ void game_round_end()
                 BG_POINT bottom_point = {6, 31};
                 main_bg_se_fill_rect_with_se(main_bg_se_get_se(bottom_point), bottom_rect);
 
-                tte_printf("#{P:%d, %d; cx:0x%X000}Cash Out: $%d", CASHOUT_RECT.left, CASHOUT_RECT.top, TTE_WHITE_PB, hands + blind_get_reward(current_blind)); // Print the cash out amount
+                int cashout_amount = hands + blind_get_reward(current_blind) + calculate_interest_reward();
+                if (cashout_amount < 10) // If cashout is two digits, omit the space so it fits in the box without clipping out
+                {
+                    tte_printf("#{P:%d, %d; cx:0x%X000}Cash Out: $%d", CASHOUT_RECT.left, CASHOUT_RECT.top, TTE_WHITE_PB, cashout_amount); 
+                }
+                else
+                {
+                    tte_printf("#{P:%d, %d; cx:0x%X000}Cash Out:$%d", CASHOUT_RECT.left, CASHOUT_RECT.top, TTE_WHITE_PB, cashout_amount); 
+                }
```

Comment:
```suggestion
                bool omit_space = cashout_amount >= 10;
                tte_printf("#{P:%d, %d; cx:0x%X000}Cash Out:%s$%d", CASHOUT_RECT.left, CASHOUT_RECT.top, TTE_WHITE_PB, omit_space ? "" : " " , cashout_amount);
```

It's cleaner to put the conditional inside the printf.